gitlab/client.py

```diff
@@ -189,6 +189,8 @@ def __init__(
         """See :class:`~gitlab.v4.objects.PagesDomainManager`"""
         self.user_activities = objects.UserActivitiesManager(self)
         """See :class:`~gitlab.v4.objects.UserActivitiesManager`"""
+        self.user_runners = objects.UserRunnerManager(self)
```

Comment:
So I don't think this should be added here. I think it should be done how `UserEmail` is done.

Reply:
See https://github.com/python-gitlab/python-gitlab/pull/2635#discussion_r1299316773 - @lgatellier I think John here meant `CurrentUserEmail` and other endpoints related to the authenticated user:

https://github.com/python-gitlab/python-gitlab/blob/d054917ccb3bbcc9973914409b9e34ba9301663a/gitlab/v4/objects/users.py#L128-L135

Reply:
@lgatellier I tend to agree with John here we like to group our manages and objects according to how GitLab structures its REST API endpoints.

But I understand this is frustrating because it becomes a 2-way step now (create the runner via the user endpoint, then lazily get the created runner for manipulation via the root endpoint). I'm also unsure why the endpoint is done like that upstream as it will be inconsistent with the other runner requests now :confused:

Reply:
What @nejch said 🙂

Reply:
Thanks @nejch  @JohnVillalovos  for your feedbacks !

I applied your suggestions. :wink: